content/sensu-go/5.2/installation/auth.md

```diff
@@ -0,0 +1,442 @@
+  "metadata": {
+    "name": "default"
+  }
+}
+{{< /highlight >}}
+
+## LDAP specification
+
+### Top-level attributes
+
+type         | 
+-------------|------
+description  | Top-level attribute specifying the [`sensuctl create`][sc] resource type. LDAP definitions should always be of type `ldap`.
+required     | Required for LDAP definitions in `wrapped-json` or `yaml` format for use with [`sensuctl create`][sc].
+type         | String
+example      | {{< highlight shell >}}"type": "ldap"{{< /highlight >}}
+
+api_version  | 
+-------------|------
+description  | Top-level attribute specifying the Sensu API group and version. For LDAP definitions, this attribute should always be `enterprise/v2`.
+required     | Required for LDAP definitions in `wrapped-json` or `yaml` format for use with [`sensuctl create`][sc].
+type         | String
+example      | {{< highlight shell >}}"api_version": "enterprise/v2"{{< /highlight >}}
+
+metadata     | 
+-------------|------
+description  | Top-level map containing the LDAP definition `name`, usually `default`. See the [metadata attributes reference][8] for details.
+required     | Required for LDAP definitions in `wrapped-json` or `yaml` format for use with [`sensuctl create`][sc].
+type         | Map of key-value pairs
+example      | {{< highlight shell >}}
+"metadata": {
+  "name": "default"
+}
+{{< /highlight >}}
+
+spec         | 
+-------------|------
+description  | Top-level map that includes the LDAP [spec attributes][sp].
+required     | Required for LDAP definitions in `wrapped-json` or `yaml` format for use with [`sensuctl create`][sc].
+type         | Map of key-value pairs
+example      | {{< highlight shell >}}
+"spec": {
+  "servers": [
+    {
+      "host": "127.0.0.1",
+      "binding": {
+        "user_dn": "cn=binder,dc=acme,dc=org",
+        "password": "P@ssw0rd!"
+      },
+      "group_search": {
+        "base_dn": "dc=acme,dc=org"
+      },
+      "user_search": {
+        "base_dn": "dc=acme,dc=org"
+      }
+    }
+  ]
+}
+{{< /highlight >}}
+
+### Spec attributes
+
+| servers    |      |
+-------------|------
+description  | An array of [LDAP servers](#server-attributes) for your directory. During the authentication process, Sensu attempts to authenticate using each LDAP server in sequence.
+required     | true
+type         | Array
+example      | {{< highlight shell >}}
+"servers": [
+  {
+    "host": "127.0.0.1",
+    "binding": {
+      "user_dn": "cn=binder,dc=acme,dc=org",
+      "password": "P@ssw0rd!"
+    },
+    "group_search": {
+      "base_dn": "dc=acme,dc=org"
+    },
+    "user_search": {
+      "base_dn": "dc=acme,dc=org"
+    }
+  }
+]
+{{< /highlight >}}
+
+<a name="groups-prefix">
+
+| groups_prefix |   |
+-------------|------
+description  | The prefix added to all LDAP groups. Use this prefix when integrating LDAP groups with Sensu RBAC [role bindings][12] and [cluster role bindings][13].
+required     | false
+type         | String
+default      | `"ldap"`
```

Comment:
While adding support for sensuctl, I realized we can't really configure any default values for `groups_prefix` & `username_prefix` because only `sensuctl create` will be supported for creation and updates (determining if the user wanted to use the default values or left them empty on purpose would be hard with this command). Probably something we will do in the web UI but for now we should remove these default values.